ehsm_kms_service/kubernetes/readme.md

```diff
@@ -0,0 +1,83 @@
+Step 1 Installing the NFS Server

```

Comment:
this file seem don't need anymore,  or you can just add a link to directly point the details to deployment_instructions.md

Reply:
It will be remove.